src/modules/shared/components/Navigation/Navigation.module.scss

```diff
@@ -1,6 +1,162 @@
+@use "sass:color";
 @use "sass:map";
 @use "styles/abstracts" as *;
 
+$c-navigation-height: 8.8rem;
+$c-navigation-border-height: 0.6rem;
+$c-navigation-list-divider-color: #4E4E4E;
+
+@mixin navigation-link-border-decoration($_offset1: 0, $_offset2: 0) {
+	$offset1-border: if($_offset1 == 0, 0, calc($_offset1 + 0.1rem));
+
+	.c-navigation__border-decoration:first-of-type {
+		&::before,
+		&::after {
+			content: "";
+			left: $_offset1;
+		}
+
+		&::after {
+			content: "";
+			left: $offset1-border;
+		}
+	}
+
+	@if $_offset2 != 0  {
+		$offset2-border: if($_offset2 == 0, 0, calc($_offset2 + 0.1rem));
+
+		.c-navigation__border-decoration:last-of-type {
+			&::before,
+			&::after {
+				content: "";
+				left: $_offset2;
+			}
+
+			&::after {
+				content: "";
+				left: $offset2-border;
+			}
+		}
+	}
+}
+
 .c-navigation {
-	z-index: get-z-layer("base");
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	height: $c-navigation-height;
+	padding: 0 3.2rem;
+	background-color: $black;
+	color: $white;
+}
+
+/**
+ * Elements
+ */
+
+.c-navigation__section {
+	display: flex;
+}
```

Comment:
Wat is het voordeel van `.c{...} .c__element{...}` over `.c{... &__element{...}}`?

Reply:
Vindbaarheid binnen de code, als je een snelle search doet en het block level element meegeeft gaat hij dat niet direct vinden met `&__element`, Je hebt ook minder nesting dus voor leesbaarheid kan daar ook iets gezegd worden maar dit is vooral preference.